### ISSUE TITLE
chapter-09 not working

### ISSUE THREAD

**Issue by jcaberio** [book-of-kubernetes/examples#14]
I followed the steps [here ](https://github.com/book-of-kubernetes/examples/tree/main/chapter-09) for aws setup. I was able to follow along from chapter 1 to chapter 8 but in chapter 9 I was unable to deploy the pods

```
ansible-playbook aws-setup.yaml

ansible-playbook playbook.yaml

./aws-ssh.sh host01

sudo su -
```
<img width="427" alt="nodes" src="https://user-images.githubusercontent.com/21195237/215307812-ad90f138-554c-45ae-822f-691db981cfcd.PNG">

<img width="790" alt="pods" src="https://user-images.githubusercontent.com/21195237/215307821-73a84c47-d672-4e8f-b385-972cc7e630a2.PNG">

**Comment by AlanHohn** [comment]
You're right to suspect an issue with the nodes. It's showing that the `kubelet` process on the nodes isn't reporting in to the API server. Did both Ansible playbooks complete successfully?

To diagnose this kind of thing, you can look at the logs for the `kubelet` service running on any of the nodes. You can do this using `journalctl -xu kubelet` as root on that node. You may find that the AWS Network Load Balancer didn't get set up correctly for some reason, and therefore traffic isn't being routed from the HA IP to an API server instance. Or, there may have been an issue with registering the nodes with the API server using the bootstrap token.
